ui/inspectors/share.reel/smb-share.reel/smb-share.html

```diff
@@ -98,10 +98,11 @@
             }
         },
         "vfsObjects": {
-            "prototype": "blue-shark/ui/field-text-input.reel",
+            "prototype": "blue-shark/ui/field-multiple-select.reel",
             "properties": {
                 "element": {"#": "vfsObjects"},
-                "label": "VFS objects"
+                "label": "VFS objects",
+                "placeholderValue": "vfs value"
             },
             "bindings": {
                 "value": {"<->": "@owner.properties.vfs_objects"}
```

Comment:
`multiple-select` component has no `value` property, it uses `values` to expose the array of values.